ci-operator/step-registry/openshift/e2e/openstack/sriov/openshift-e2e-openstack-sriov-workflow.yaml

```diff
@@ -11,6 +11,7 @@ workflow:
       - ref: optional-operators-subscribe
       - ref: openstack-provision-sriov-networknodepolicy
     test:
+      - ref: openstack-test-dpdk
```

Comment:
Having multiple steps in the `test` stanza means that if one fails, the next ones are not run. Are you OK with that?

Reply:
yes

Reply:
I guess we need to remove the `openstack-test-dpdk` step from the [`openshift-e2e-openstack-nfv` workflow](https://github.com/openshift/release/blob/41439f3/ci-operator/step-registry/openshift/e2e/openstack/nfv/openshift-e2e-openstack-nfv-workflow.yaml#L15) otherwise it would run twice, right?